test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -138,12 +138,17 @@ test_ucp_peer_failure::am_callback(void *arg, const void *header,
     return UCS_OK;
 }
 
-void test_ucp_peer_failure::err_cb(void *arg, ucp_ep_h ep, ucs_status_t status) {
+void test_ucp_peer_failure::err_cb(void *arg, ucp_ep_h ep, ucs_status_t status)
+{
     test_ucp_peer_failure *self = reinterpret_cast<test_ucp_peer_failure*>(arg);
     EXPECT_TRUE((UCS_ERR_CONNECTION_RESET == status) ||
                 (UCS_ERR_ENDPOINT_TIMEOUT == status));
     self->m_err_status = status;
     ++self->m_err_count;
+
+    entity *e = self->find_entity_by_ep(ep);
+    ASSERT_TRUE(e != NULL);
```

Comment:
is this still OK or need to check if e!=NULL?

Reply:
I think it worth having ASSERT here to make sure that entity is not NULL.
but it has to be non-NULL with current tests.
if new tests are added, they will hit this assertion in case of something is wrong.

Reply:
ok